sdks/python/apache_beam/runners/runner_test.py

```diff
@@ -51,21 +61,65 @@ def test_create_runner(self):
   def test_remote_runner_translation(self):
     remote_runner = DataflowPipelineRunner()
     p = Pipeline(remote_runner,
-                 options=PipelineOptions([
-                     '--dataflow_endpoint=ignored',
-                     '--job_name=test-job',
-                     '--project=test-project',
-                     '--staging_location=ignored',
-                     '--temp_location=/dev/null',
-                     '--no_auth=True'
-                 ]))
+                 options=PipelineOptions(self.default_properties))
 
     (p | 'create' >> ptransform.Create([1, 2, 3])  # pylint: disable=expression-not-assigned
      | 'do' >> ptransform.FlatMap(lambda x: [(x, x)])
      | 'gbk' >> ptransform.GroupByKey())
     remote_runner.job = apiclient.Job(p.options)
     super(DataflowPipelineRunner, remote_runner).run(p)
 
+  def test_remote_runner_display_data(self):
+    remote_runner = DataflowPipelineRunner()
+    p = Pipeline(remote_runner,
+                 options=PipelineOptions(self.default_properties))
+
+    class SpecialParDo(beam.ParDo):
```

Comment:
Usually we don't want to be subclassing beam.ParDo. We should instead make beam.ParDo do the right thing (as well as all the other transforms). This is probably a follow-up PR, so for now make this a custom PTransform that invokes ParDo in its apply instead.